server/test/services/applicant/question/PhoneQuestionTest.java

```diff
@@ -92,7 +92,7 @@ public void withInvalidApplicantData_missingRequiredFields() {
   }
 
   @Test
-  @Parameters({"5552123333", "123asc1234"})
+  @Parameters({"5552123333", "1231234567"})
```

Comment:
Question (blocking): Is looks like this *is* a valid phone number? Is this not a valid phone number for some reason?

Reply:
both are invalid. "The short answer is that most 555 numbers are not working numbers, so real people won't be harassed if diehard fans try to call them. "

Reply:
Got it thanks!